src/components/title/_fields/publication-type/publication-type.js

```diff
@@ -13,7 +14,7 @@ export default function PublicationTypeField() {
       <Field
         name="publicationType"
         component={Select}
-        label="Publication type"
+        label={intl.formatMessage({ id: 'ui-eholdings.title.publicationType' })}
         dataOptions={[
           { value: 'Audio Book', label: 'Audio Book' },
```

Comment:
On Publication Type data -- depending on spike outcome for localization of filters -- we may want to translate these as well